tools/github/autoresponder.ts

```diff
@@ -120,7 +122,32 @@ console.log("similar discussions:", similarDiscussions);
 const answerFromAi = await getAnswerFromAi(API_SECRET, DISCUSSION_BODY);
 console.log("response from AI:", answerFromAi.trim().slice(0, 300));
 
-const fullBody = `${commentHeader}${answerFromAi}\n\n---\n\nHere are some similar discussions:\n\n${similarDiscussions}`;
+const hasAnswerFromAi = !answerFromAi.includes("Sorry, I don't know how to help with that.");
+const hasSimilarDiscussions = similarDiscussions.length > 0 && similarDiscussions !== "No similar discussions found.";
+
+// if you don't have an answer from AI AND you don't have similar discussions, don't bother commenting
+if (!hasSimilarDiscussions && !hasAnswerFromAi) {
+  // @ts-expect-error deno
+  Deno.exit(0);
+}
+
+let fullBody = `${commentHeader}`;
+
+// answer and similar discussions
+if (hasAnswerFromAi && hasSimilarDiscussions) {
+  const similarDiscussionPrefix =
+    "\n\nI also did a search, and I managed to find these other Discussions that might be similar or related to your question:\n\n";
```

Comment:
An additional sentence like "Give them a read to see if they answer your question. If they do, please mark this as the answer to your discussion." Might get them all the way there to actually clicking on the link. And if we're not asking them to reply or close/answer the discussion, we'd have no way of knowing that they don't need our help anymore, and we'd still spend time writing an answer.